app/assets/javascripts/assignments.js

```diff
@@ -250,10 +262,39 @@
     $(".file-picker").each(function(index) { activate_file_picker($(this)); });
   }
 
+  function toggles_init() {
+    $.each($('.toggle-wrapper'), function(index, toggleWrapper) {
+      $(toggleWrapper).click(function(e) {
```

Comment:
Also, since these are checkboxes, I think you want the event handler to be `.change`, rather than `.click` -- that way, you'll get keyboard-driven changes too.

Reply:
The problem with `change` is that it gets called when I change the toggle inside the wrapper (if you try it you'll see the buttons flip back and forth forever)